frontend/components/TimeSelect.vue

```diff
@@ -0,0 +1,54 @@
+<!--
+時間選択ダイアログ
+v-model: 時間
+-->
+<template>
+  <v-dialog v-model="dialog" max-width="300px">
+    <template #activator="on">
+      <slot name="activator" :on="{ click: open }"></slot>
+    </template>
+    <v-card>
+      <v-time-picker v-model="editTime" format="ampm" full-width>
```

Comment:
AMとPMどっちが午前か分からなくなる人もいるので24時間表記にした方が分かりやすいかと思いました。
```suggestion
      <v-time-picker v-model="editTime" format="24hr" full-width>
```

Reply:
24時間表示に変更しました。